scripts/evaluate_predicate_classifiers.py

```diff
@@ -0,0 +1,140 @@
+"""Script to evaluate learned predicate classifiers on held-out test cases."""
+
+import os
+from typing import List, Optional, Set
+
+import numpy as np
+
+from predicators.src import utils
+from predicators.src.approaches import create_approach
+from predicators.src.approaches.bilevel_planning_approach import \
+    BilevelPlanningApproach
+from predicators.src.envs import BaseEnv, create_new_env
+from predicators.src.main import _generate_interaction_results, \
+    _generate_or_load_offline_dataset
+from predicators.src.settings import CFG
+from predicators.src.structs import Action, Dataset, Predicate, Task
+from predicators.src.teacher import Teacher
+
+
+def _main() -> None:
+    # Parse & validate args
```

Comment:
see my comment below first, and then after that, I'm wondering how much of this function can potentially be removed

Reply:
if i make this specific to InteractiveLearningApproach, most of it can go away right? we don't need to seed or anything since we're just loading every cycle

Reply:
actually it doesn't need to be specific to InteractiveLearningApproach